libraries/Ticker/Ticker.h

```diff
@@ -34,17 +36,22 @@ class Ticker
 public:
 	Ticker();
 	~Ticker();
-	typedef void (*callback_t)(void);
+
 	typedef void (*callback_with_arg_t)(void*);
+	typedef std::function<void(void)> TickerFunction;
 
-	void attach(float seconds, callback_t callback)
+	void attach(float seconds, TickerFunction tf, bool reqSchedule = false)
```

Comment:
Also, why did `void attach(float seconds, void (*callback)(TArg), TArg arg)` overload not get the version with "scheduled" option?

I would expect to see the product of {attach, once} x {sec, msec} x {no arg, arg} x {normal, scheduled}, i.e. 16 functions.

Reply:
There's no point in repeatedly `attach` the same callback as the callback will run after the loop and there no way (yet?) to stop that. Or do I miss something?

Just asking because I'm using the esp8266's interface in my esp32's version of the Ticker lib.